### PR TITLE
[feat/aut896] Adding issue template - bug report

### PR DESCRIPTION
**Pull request by bertrandboudaud**
This is the template when user want to fill a bug report.

This is based on the template that github propose for bug report, with some minor modifications.

Note that github proposes 2 kinds of templates: bug report or features request.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**logs**
+Attach log file. Log, by default, can be found in:
+ - Windows: C:\Users\user_name\AppData\Local\SmartPeak
+ - Linux: /home/user_name/.SmartPeak
+ - MacOS: /Users/user_name/.SmartPeak
```
Comment (bertrandboudaud) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: @ahmedskhalil can you please confirm this will be the default directory for MacOS?

**Comment by ChristopherAbram** [comment]
Bug report template looks nice. Don't we need feature request template?

**Review by ChristopherAbram — APPROVED** [review]

**Review by bertrandboudaud — COMMENTED** [review]
> Bug report template looks nice. Don't we need feature request template?

probably yes, but we haven't talked about that. maybe better to talk this all together as it impacts our way to communicate, we must be all agreed.
(I am for it also)

**Review by ghost — APPROVED** [review]
looking nice

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ghost) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: @bertrandboudaud yes this corresponds to the user's default directory on macOS

**Review by dmccloskey — APPROVED** [review]

**Pull request merged**